portal/server/routes/contract/about/supplierPage.js

```diff
@@ -58,7 +56,19 @@ router.get('/contract/:_id/about/supplier', [validateRole({ role: [MAKER] }), pr
     };
     req.session.aboutSupplierFormData = null;
   }
-
+  if(deal.submissionDetails['supplier-companies-house-registration-number']){
```

Comment:
judging by our e2e spec, we do not need to include this here (we already have a passing test for 'A maker picks up a deal in status=Draft, triggers Supplier companies house validation errors'). Either our test is not working and this change is needed (in which case I'd also expect a test rewrite) or this is not needed here?